src/latch/__init__.py

```diff
@@ -27,6 +27,7 @@
     g6e_12xlarge_task,
     g6e_16xlarge_task,
     g6e_24xlarge_task,
+    g6e_48xlarge_task,
```

Comment:
this is fine but for the future lets stop adding imports into the __init__ - will be removing these eventually and dont want the migration to be too bad